src/licensedcode/data/rules/gpl-3.0_19.yml

```diff
@@ -0,0 +1,3 @@
+licenses:
+    - gpl-3.0

```

Comment:
This is not correct: there should be two licenses: gpl-3.0 plus the other one which is an exception of sorts